plugins.json

```diff
@@ -1,4 +1,14 @@
 {
+    "core": {
+        "name": "aiida-core",
+        "package_name": "aiida_core",
+        "entry_point": "",
+        "state": "stable",
+        "pip_url": "git+https://github.com/aiidateam/aiida-core",
+        "plugin_info": "https://raw.githubusercontent.com/aiidateam/aiida-core/v1.0.0b6/setup.json",
```

Comment:
the current `master` does not have a `setup.json`.
I think the last beta tag is currently the best solution.
If you agree, I'll open an issue to switch this back to `master` once 1.0 is released.